lips/lip-8.md

```diff
@@ -148,7 +226,7 @@ Valid values are:
 * `rejected` - User/VASP did not approve the pre-approval request.
 * `closed` - Approval has been closed by the user/VASP and can no longer be used.
 
-**Valid Status Transitions**. Either party in the pre-approval agreement can mutate the status. The status always initially begins as `pending` at which time a user must agree to the pre-approval request.  Once the user has reviewed the request, the billee VASP will update the pre-approval status to `valid` (if the user agreed) or `rejected` (if the user rejected the pre-approval).
+**Valid Status Transitions**. Either party in the pre-approval agreement can mutate the status. The status always initially begins as `pending` at which time a user must agree to the pre-approval request.  Once the user has reviewed the request, the biller VASP will update the pre-approval status to `valid` (if the user agreed) or `rejected` (if the user rejected the pre-approval).
```

Comment:
This would still be billee.  Or maybe we should user "payer" and "payee" instead.  In which case, this would be the payer VASP.  The side that has the user on it.  Actually, let's just call it the sender VASP (buyer) maybe